StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -66,20 +62,20 @@ public extension PaymentSheet {
         ///   - onBehalfOf: The account (if any) for which the funds of the intent are intended
         ///   - paymentMethodConfigurationId: Configuration ID (if any) for the selected payment method configuration
         ///   - confirmHandler: A handler called with payment details when the user taps the primary button (e.g. the "Pay" or "Continue" button).
-        ///   - isCVCRecollectionEnabledCallback: Callback to determine whether to display the CVC recollection form
+        ///   - requireCVCRecollection: If true, PaymentSheet recollects CVC for saved cards before confirmation (PaymentIntent only)
         @_spi(EarlyAccessCVCRecollectionFeature)
         public init(mode: Mode,
                     paymentMethodTypes: [String]? = nil,
                     onBehalfOf: String? = nil,
                     paymentMethodConfigurationId: String? = nil,
                     confirmHandler: @escaping ConfirmHandler,
-                    isCVCRecollectionEnabledCallback: CVCRecollectionEnabledCallback? = nil) {
+                    requireCVCRecollection: Bool = false) {
             self.mode = mode
             self.paymentMethodTypes = paymentMethodTypes
             self.onBehalfOf = onBehalfOf
             self.paymentMethodConfigurationId = paymentMethodConfigurationId
             self.confirmHandler = confirmHandler
-            self.isCVCRecollectionEnabledCallback = isCVCRecollectionEnabledCallback ?? { return false }
+            self.requireCVCRecollection = requireCVCRecollection
             validate()
         }
```

Comment:
We can delete this once we remove `EarlyAccessCVCRecollectionFeature` right?

Reply:
yep!